compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -123,7 +123,6 @@ abstract class GenJSCode extends plugins.PluginComponent
 
     override def name: String = phaseName
     override def description: String = GenJSCode.this.description
-    override def erasedTypes: Boolean = true
```

Comment:
I had to add this change to support the new candidate for 2.13.0-M5, i.e., 2.13.0-pre-8a52aa1. It turns out that the inherited value of `erasedTypes` is already always `true` (across all versions of Scala we support) so this line has always been redundant.